File: javascript/1114.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

while(lines.length){
    senha=parseInt(lines.shift())
    if(senha == 2002){
        console.log("Acesso Permitido")
        break
    }else{
        console.log("Senha Invalida")
    }
}

File: C/1114.c
#include <stdio.h>

int main(){
        
    int token=0,senha;
    while(!token){
        scanf("%i",&senha);
        if(senha == 2002){
            token=1;
            printf("Acesso Permitido\n");
        }else printf("Senha Invalida\n");
    }

    return 0;
}